src/widget/wtracktableview.cpp

```diff
@@ -449,6 +449,15 @@ void WTrackTableView::slotPurge() {
     }
 }
 
+void WTrackTableView::slotDeleteTracksFromDisk() {
+    QModelIndexList indices = selectionModel()->selectedRows();
+
+    if (indices.size() > 0) {
```

Comment:
!indices.isEmpty()

Reply:
:+1: 
for sake of completeness I replaced all occurences in this file with an early return
```
if (indices.isEmpty()) {
    return;
}
```